server/protocols/webadmin/webadmin-jmap/src/main/java/org/apache/james/webadmin/data/jmap/RecomputeAllFastViewTaskAdditionalInformationDTO.java

```diff
@@ -32,42 +34,57 @@
     public static final AdditionalInformationDTOModule<RecomputeAllFastViewProjectionItemsTask.AdditionalInformation, RecomputeAllFastViewTaskAdditionalInformationDTO> SERIALIZATION_MODULE =
         DTOModule.forDomainObject(RecomputeAllFastViewProjectionItemsTask.AdditionalInformation.class)
             .convertToDTO(RecomputeAllFastViewTaskAdditionalInformationDTO.class)
-            .toDomainObjectConverter(dto -> new RecomputeAllFastViewProjectionItemsTask.AdditionalInformation(
-                dto.getProcessedUserCount(),
-                dto.getProcessedMessageCount(),
-                dto.getFailedUserCount(),
-                dto.getFailedMessageCount(),
-                dto.timestamp))
-            .toDTOConverter((details, type) -> new RecomputeAllFastViewTaskAdditionalInformationDTO(
-                type,
-                details.timestamp(),
-                details.getProcessedUserCount(),
-                details.getProcessedMessageCount(),
-                details.getFailedUserCount(),
-                details.getFailedMessageCount()))
+            .toDomainObjectConverter(RecomputeAllFastViewTaskAdditionalInformationDTO::toDomainObject)
+            .toDTOConverter(RecomputeAllFastViewTaskAdditionalInformationDTO::toDTO)
```

Comment:
related?

Reply:
Yes it enhence readability via method extraction and is much needed